client/my-sites/earn/home.tsx

```diff
@@ -262,6 +263,51 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 		};
 	};
 
+	/**
+	 * Return the content to display in the Paid Newsletter card based on the current plan.
+	 *
+	 * @returns {object} Object with props to render a PromoCard.
+	 */
+	const getPaidNewsletterCard = () => {
+		const cta = isFreePlan
+			? {
+					text: translate( 'Unlock this feature' ),
+					action: () => {
+						trackUpgrade( 'any-paid-plan', 'premium-content' );
+						page( `/plans/${ selectedSiteSlug }` );
+					},
+			  }
+			: {
+					text: translate( 'Learn how to get started' ),
+					action: () => {
+						trackCtaButton( 'learn-paid-newsletters' );
+						page( `https://wordpress.com/support/paid-newsletters/` );
```

Comment:
I think this should be wrapped in `localizeUrl`, and can use standard ticks (`'`), as there's no interpolation required.

Reply:
@kwight I just tried that in my local environment and it pushes the users to a non-existent page. Is there a way to use localizeUrl so that defaults to the EN page if the translation isn't available?

Reply:
In the contextual help page it seems that they make that [`localizeUrl` call](https://github.com/Automattic/wp-calypso/blob/master/client/blocks/inline-help/contextual-help.js) to similar support pages...

Reply:
If there's only an EN page let's file an issue to localize the support page and update this as a follow up. I don't think this is blocking.

Reply:
@gwwar where should we file an issue to get the support page localized? cc @lauac

Reply:
It'd mostly be a reminder for us to do, so wp-calypso is fine. We'd likely chat with team global or happiness on how the process normally goes to queue this up.

Reply:
> Is there a way to use localizeUrl so that defaults to the EN page if the translation isn't available?

@donlair Hm, not that I know of. I'd say we trust `localizeURl` to figure out the appropriate URL based on the user's set interface language in `/me`, whether the page exists or not; non-existant pages will include a prompt and link directing the user to the English version if they wish.

<img width="782" alt="Screen Shot 2020-06-08 at 2 19 18 PM" src="https://user-images.githubusercontent.com/349751/84081602-238dbe80-a993-11ea-9eb4-b0f83e842a04.png">